rotkehlchen/db/accounting_rules.py

```diff
@@ -517,7 +518,7 @@ def query_missing_accounting_rules(
                 event_identifiers=list({event.event_identifier for event in events}),
                 order_by_rules=[('event_identifier', True), ('sequence_index', True)],
             ),
-            has_premium=True,
+            entries_limit=INTERNAL_QUERY_LIMIT,
```

Comment:
same. In some places we used has_premium=True because we need to scan all the events but we don't return everything to the user later

Reply:
I would say the cleanest solution is to just separate into internal and external use functions. Just have one without any arguments which just gets no limits.